run.py

```diff
@@ -65,6 +65,7 @@
         [--osp-cred <file>]
         [--rhs-ceph-repo <repo>]
         [--ubuntu-repo <repo>]
+        [--use-local-rhel-repos]
```

Comment:
IMHO, this is specific to `install_prereq` module and should be passed to it. I don't see it's usage across the framework.

Reply:
`run.py` already have similar options (e.g `--skip-subscription`, `--add-repo` etc), adding this as custom-config option unnecessarily add complexity and no any config passed through test to install-prereq.